cmd/editorconfig/main.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"bytes"
+	"path/filepath"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+
+	"gopkg.in/ini.v1"
+
+	ec "github.com/editorconfig/editorconfig-core-go"
+)
+
+const (
+	Version = "1.2.0"
+)
+
+func main() {
+	var (
+		configname string
+		configversion string
+		showVersionFlag bool
+	)
+	flag.StringVar(&configname, "f", ec.ConfigNameDefault, "Specify conf filename other than `.editorconfig'")
+	flag.StringVar(&configversion, "b", "", "Specify version (used by devs to test compatibility)")
+	flag.BoolVar(&showVersionFlag, "v", false, "Display version information")
+	flag.BoolVar(&showVersionFlag, "version", false, "Display version information")
+	flag.Parse()
+
+	if showVersionFlag {
+		fmt.Printf("EditorConfig Core Go, Version %s\n", Version)
+		os.Exit(0)
+	}
+
+	rest := flag.Args()
+
+	for _, file := range rest {
+		if !filepath.IsAbs(file) {
```

Comment:
Why this requirement?

You can just use `filepath.Abs` to convert a path to absolute (it will just return itself if already absolute)

Reply:
This is the same behavior as editorconfig-core-c does.

I agree that this is a bit too strict rule, but I think core executables should behaves similarly for this so I implemented in this way for now.